feisty/core/interface.py

```diff
@@ -342,8 +359,10 @@ def _compute_predation(self):
             self.biomass,
             self.food_web,
         )
-        self.tendency_data.predation_rate[:, :] = (
-            self.tendency_data.predation_flux.data / self.biomass.isel(group=self.ndx_fish).data
+        self.tendency_data.predation_rate[
+            :, :
+        ] = self.tendency_data.predation_flux.data / np.maximum(
+            self.biomass.data[self.ndx_fish, :], 1e-300
```

Comment:
should `1e-300` be replaced by `eps`?

Perhaps we should also introduce a parameter for minimum biomass?